modules/softwarelandscape/exercise2.md

```diff
@@ -4,44 +4,14 @@ type: exercise
 order: 3
 ---
 
-## Exercise: Mapping the principles
+## **Exercise:** Mapping the principles
 
-### The FAIR4RS Principles are:
+Choose one of the three examples of software below. They have been described with statements that can be mapped to the FAIR4RS Principles. With each statement in the software description, identify the principle(s) it maps to, and the (sub)facet(s) of the principle(s) it addresses. A single statement may map to multiple (sub)facets.
 
-#### F: Software, and its associated metadata, is easy for both humans and machines to find.
+![FAIR for Research Software principles](media/fair4rs_principles.png)
```

Comment:
Is there a way to have the deleted text as alt text for this image? Just thinking about screen readers or something not being able to parse the image so easily. I do personally find the image easier and more pleasant to read, though.

Reply:
We will change this into a table, an issue was opened: https://github.com/esciencecenter-digital-skills/research-software-support/issues/124